zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepo.java

```diff
@@ -88,7 +88,7 @@ protected void setNotebookDirectory(String notebookDirPath) throws IOException {
       LOGGER.info("Notebook dir doesn't exist: {}, creating it.",
           rootNotebookFileObject.getName().getPath());
     }
-    this.rootNotebookFolder = rootNotebookFileObject.getName().getPath();
+    this.rootNotebookFolder = rootNotebookFileObject.getName().getURI().replace("file:///", "/");
```

Comment:
Do you mind to add some comment on this change to help others understand why call `getURI().replace("file:///", "/")`?

Reply:
Done

Reply:
Thanks @Muhammad-ms LGTM